### PR TITLE
fix licensing headers

### LINKED ISSUE #1383: Change file headers from cpp-ethereum to solidity
Currently it says *This file is part of cpp-ethereum.*, but it should read *Solidity*.

We should also probably settle on a style regarding comments:
```
/*
        This file is part of cpp-ethereum.

        cpp-ethereum is free software: you can redistribute it and/or modify
        it under the terms of the GNU General Public License as published by
        the Free Software Foundation, either version 3 of the License, or
        (at your option) any later version.

        cpp-ethereum is distributed in the hope that it will be useful,
        but WITHOUT ANY WARRANTY; without even the implied warranty of
        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
        GNU General Public License for more details.

        You should have received a copy of the GNU General Public License
        along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
*/
/**
 * @author Christian <c@ethdev.com>
 * @author Gav Wood <g@ethdev.com>
 * @date 2014
 * Unit tests for the solidity expression compiler, testing the behaviour of the code.
 */

```

### PR DESCRIPTION
**Pull request by VoR0220**
Signed-off-by: VoR0220 <rj@erisindustries.com>

fixes #1383

### CODE REVIEW

**Comment by chriseth** [comment]
You accidentally added some binaries

File: scripts/install_deps.sh [hunk 1]
```diff
@@ -110,8 +110,8 @@ case $(uname -s) in
         # the bottle is known broken, so we will just cheat and use a hardcoded ZIP for
         # the time being, which is good enough.   The cause of the breaks will go away
         # when we commit the repository reorg changes anyway.
-        curl -L -O https://github.com/bobsummerwill/cpp-ethereum/releases/download/v1.3.0/cpp-ethereum-osx-mavericks-v1.3.0.zip
```
Comment (chriseth) [inline scripts/install_deps.sh:113]: This should not be changed.

File: scripts/install_eth.cmake [hunk 2]
```diff
@@ -73,4 +73,4 @@ endfunction(download_and_unpack)
 get_filename_component(ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}/.." ABSOLUTE)
 set(CACHE_DIR "${ROOT_DIR}/deps/cache")
 set(INSTALL_DIR "${ROOT_DIR}/deps/install/x64/eth")
-download_and_unpack("https://github.com/bobsummerwill/cpp-ethereum/releases/download/develop-v1.3.0.401/cpp-ethereum-develop-windows.zip" ${INSTALL_DIR})
```
Comment (chriseth) [inline scripts/install_eth.cmake:76]: No change here either.

File: test/libsolidity/SolidityNameAndTypeResolution.cpp [hunk 3]
```diff
@@ -1051,7 +1051,7 @@ BOOST_AUTO_TEST_CASE(missing_state_variable)
 
 BOOST_AUTO_TEST_CASE(base_class_state_variable_accessor)
 {
-	// test for issue #1126 https://github.com/ethereum/cpp-ethereum/issues/1126
```
Comment (chriseth) [inline test/libsolidity/SolidityNameAndTypeResolution.cpp:1054]: Please don't change.

**Review by chriseth — CHANGES_REQUESTED** [review]
Please remove binaries and rebase.

**Comment by VoR0220** [comment]
Crap. My bad. Will change.

**Comment by chriseth** [comment]
Superseded by https://github.com/ethereum/solidity/pull/1425

**Pull request closed without merging**